### ISSUE TITLE
Drop automatic users and groups backups

### ISSUE THREAD

**Issue by cmb69** [cmb69/register_xh#62]
A single backup for each modification of the users or groups file only would make sense to restore after an erroneous bulk ~~backup~~update from the administration. However, before the file is restored, another user may already have triggered an update of the file, and a new backup would have been created, overwriting the backup the admin would like to restore. Without careful checking (or protections, such as maintainance mode), they would not restore the desired state, but would even overwrite the update of that other user.

A more elaborate backup strategy might make sense, but I consider it out of scope due to more important needs for this plugin.

Thus I propose to drop the automatic users and groups backups altogether.

PS: The potentional need to have a backup of the users/groups are the full file updates in the adminstration. These should not be possible at all, at least not without an offline look and some form of conflict resolution UI. IIRC, svasti did an interesting variant for Memberpages (where only modified records would be stored). Another option would be a classic CRUD UI.

**Comment by cmb69** [comment]
Well, this is not urgent, and I still don't have the time to work on a better solution, so let's postpone this.

**Comment by cmb69** [comment]
After #68, the backups make even less sense then before, so we're going to drop them without replacement (user should backup `content/` anyway from time to time).
